client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize-conflicts.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { translate } from 'i18n-calypso';
+
+class DomainConnectAuthorizeConflicts extends Component {
+	placeholder = () => {
+		return (
+			<div>
+				<div className="domain-connect__is-placeholder">
+					<span></span>
+					<span></span>
+				</div>
+			</div>
+		);
+	}
+
+	render() {
+		const { isPlaceholder, conflictingRecords } = this.props;
```

Comment:
For some reason, I'm not a fan of the `isPlaceholder` variable name - I prefer something like `isLoading`. The reason for that is `isPlaceholder` seems to be telling the component how it should behave to the state that is passed to it, while `isLoading` is just passing the state, letting the component deal with it however it wants.
It's just a preference - feel free to use whatever convention you want, the key is to be consistent (which you are, of course).